server/api.go

```diff
@@ -31,6 +31,11 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 			Method:  http.MethodPost,
 			Handler: requiresUserId(p.submitScore),
 		},
+		{
+			Path:    "/api/v1/disable_for_user",
```

Comment:
Do we want any way to enable the survey again?

Reply:
cc @esethna

Reply:
No, probably not necessary. They can still always type to surveybot to give feedback even if automated surveys are disabled